### PR TITLE
API Generator Code for Keras

### PR DESCRIPTION
**Pull request by sampathweb**
API Generator Code.

### CODE REVIEW

File: shell/format.sh [hunk 1]
```diff
@@ -3,6 +3,10 @@ set -Eeuo pipefail
 
 base_dir=$(dirname $(dirname $0))
 
+echo "Generating public APIs..."
+python3 "${base_dir}"/api_gen.py
```
Comment (fchollet) [inline shell/format.sh:7]: Upon thinking more about it -- an issue with generating the API in this script is that it takes quite a bit of time, but most of the time it won't be needed (it's only needed when we add new APIs). So it's actually better to have it in a separate script after all.

**Review by fchollet — COMMENTED** [review]
The changes look good!

File: shell/format.sh (hunk 1 above)
Comment (sampathweb) [reply shell/format.sh]: Thanks for the feedback. Added `shell/api_gen.sh` to generate APIs and kept `shell/format.sh` for Code formatting.

**Comment by sampathweb** [comment]
Closing this PR since https://github.com/keras-team/keras/pull/19530 is now merged.

**Pull request closed without merging**